DEV_ENV.md

```diff
@@ -2,12 +2,12 @@
 
 ## Development quickstart
 
-1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`.
-1. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
-1. From the root of this repository, run `make local-init` to build and run the dev environment. The first build takes awhile, but subsequent runs will use cached artifacts.
-1. Visit [http://backend.corporanet.local:5000](http://backend.corporanet.local:5000) to view the backend, and [http://frontend.corporanet.local:3000](http://frontend:3000) for the frontend.
-1. `make local-dbconsole` starts a connection with the local postgresql db.
-1. **Open the source code and start editing!**
+1. [install docker](https://docs.docker.com/get-docker/). If brew is installed run `brew install docker`. If you have a Mac, [install Docker Desktop](https://www.docker.com/products/docker-desktop) and open it so it's running on your machine.
+2. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
```

Comment:
I think if you leave theses all as 1s markdown will auto convert them to the correct numbers and then you dont need to worry about renumbering if you add a new step

Reply:
```suggestion
1. [install chamber](https://github.com/segmentio/chamber). If brew is installed run `brew install chamber`. (This is needed for running functional tests.)
```